pkg/operator/ceph/object/user.go

```diff
@@ -88,7 +88,7 @@ func decodeUser(data string) (*ObjectUser, int, error) {
 
 // GetUser returns the user with the given ID.
 func GetUser(c *Context, id string) (*ObjectUser, int, error) {
-	logger.Infof("Getting user: %s", id)
+	logger.Infof("getting s3 user %q", id)
```

Comment:
While a small thing, how about if we refer to it as the "object user" instead of the "s3 user" so it follows the CR naming? This suggestion is not just here, but for a bunch of logging and error messages.
```suggestion
	logger.Infof("getting object user %q", id)
```

Reply:
I believe s3 user is more accurate but if we have an existing convention I'll stick with it and user `object user`.

Reply:
It's just for logging so it's fine either way actually